### DISCUSSION TITLE
[iOS] - Appium Scroll Issue in Three Layout Mode in iPad Device[Landscape]

### DISCUSSION THREAD

**Discussion by yukender** [appium/appium#19317] [Q&A]
Hi, 

We've been conducting mobile automation testing on iPad devices, specifically in [Landscape] Orientation. During these tests, we encountered an issue related to scrolling within the application interface in three layout mode.

Our objective is to scroll to the second layout containing metadata, and we have identified the element with the accessibility value "metadataTable." However, when the iPad is in Landscape mode, our Appium script erroneously scrolls to the first layout instead, causing our test cases to fail.

It's worth noting that the scrolling operation works as expected when the iPad is in Portrait mode, allowing us to access the correct element in the second layout.

**Screenshot of the three layout Mode :-** 
![MicrosoftTeams-image](https://github.com/appium/appium/assets/62298454/ba161d85-d095-4eda-bd6f-07e226ea35c7)

**Appium Inspect Value :-**
![image](https://github.com/appium/appium/assets/62298454/f4625767-4e77-4d93-8cc4-58b16ffd3015)

**TestAutomation Report :-** 
[iOSAutomationTestReport.zip](https://github.com/appium/appium/files/13024369/iOSAutomationTestReport.zip)

**Here are the details of our environment :-**

Appium Version: 1.22.2
iPad OS Version: 16.2

**Below is the scroll script that we have been using for iOS :-**
 HashMap<Object, Object> scrollObject = new HashMap<>();
 scrollObject.put("element", scrollList.getId());
 scrollObject.put("predicateString", "value == '" + expectedText + "'");
 scrollObject.put("direction", "down");
 driver.executeScript("mobile: scroll", scrollObject);

**Comment by jlipps** [comment]
I'm not sure of the internals here but I think `mobile: scroll` just calls Apple's own scroll APIs. So I'm guessing this bug is in XCUITest itself.
